src/api/app/models/event_subscription.rb

```diff
@@ -100,6 +102,11 @@ def parameters_for_notification
   def self.without_disabled_or_internal_channels
     channels.keys.reject { |channel| channel == 'disabled' || channel.in?(INTERNAL_ONLY_CHANNELS) }
   end
+
+  def track_event_subscription_enabled
+    RabbitmqBus.send_to_bus('metrics',
+                            "event_subscription.enabled,event_type=#{eventtype},channel=#{channel},value=1")
```

Comment:
`value` will be processed as a tag if the last comma isn't replaced by a space.

```suggestion
                            "event_subscription.enabled,event_type=#{eventtype},channel=#{channel} value=1")
```